README.md

```diff
@@ -30,7 +30,7 @@ The following table provides an overview of the languages supported by Kiota and
 | TypeScript/JavaScript | 🛠️ | [🛠️](https://github.com/microsoft/kiota-typescript/tree/main/packages/abstractions) | [FORM](https://github.com/microsoft/kiota-typescript/tree/main/packages/serialization/form), [JSON](https://github.com/microsoft/kiota-typescript/tree/main/packages/serialization/json), [MULTIPART](https://github.com/microsoft/kiota-typescript/tree/main/packages/serialization/multipart), [TEXT](https://github.com/microsoft/kiota-typescript/tree/main/packages/serialization/text) | [Anonymous](https://github.com/microsoft/kiota-typescript/blob/main/packages/abstractions/src/authentication/anonymousAuthenticationProvider.ts), [API Key](https://github.com/microsoft/kiota-typescript/blob/main/packages/abstractions/src/authentication/apiKeyAuthenticationProvider.ts), [Azure](https://github.com/microsoft/kiota-typescript/tree/main/packages/authentication/azure), [SPFx](https://github.com/microsoft/kiota-typescript/tree/main/packages/authentication/spfx) | [🛠️](https://github.com/microsoft/kiota-typescript/tree/main/packages/http/fetch) | [link](https://learn.microsoft.com/openapi/kiota/quickstarts/typescript) |
-| Dart | [▶](https://github.com/kiota-community/kiota-dart/tree/features/dart) | [🛠️](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_abstractions) | [🛠️ FORM](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_serialization_form), [▶ JSON](https://github.com/kiota-community/dart_kiota/pull/35), [▶ MULTIPART](https://github.com/kiota-community/dart_kiota/pull/35), [🛠️ TEXT](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_serialization_text) | [🛠️ Anonymous](https://github.com/kiota-community/dart_kiota/blob/main/packages/kiota_abstractions/lib/src/authentication/anonymous_authentication_provider.dart), [🛠️ API Key](https://github.com/kiota-community/dart_kiota/blob/main/packages/kiota_abstractions/lib/src/authentication/api_key_authentication_provider.dart) | [🛠️](https://github.com/microsoft/kiota-typescript/tree/main/packages/http/fetch) | |
+| Dart | [▶](https://github.com/kiota-community/kiota-dart/tree/features/dart) | [🛠️](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_abstractions) | [🛠️ FORM](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_serialization_form), [▶ JSON](https://github.com/kiota-community/dart_kiota/pull/35), [▶ MULTIPART](https://github.com/kiota-community/dart_kiota/pull/35), [🛠️ TEXT](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_serialization_text) | [🛠️ Anonymous](https://github.com/kiota-community/dart_kiota/blob/main/packages/kiota_abstractions/lib/src/authentication/anonymous_authentication_provider.dart), [🛠️ API Key](https://github.com/kiota-community/dart_kiota/blob/main/packages/kiota_abstractions/lib/src/authentication/api_key_authentication_provider.dart) | [🛠️](https://github.com/kiota-community/dart_kiota/tree/main/packages/kiota_http) | |
```

Comment:
languages are ordered alphabetically, can you please move Dart (and CLI as I now realize it's at the wrong place)?

Reply:
Sorted